### ISSUE TITLE
Forum Seems Incompatible With Google Translate

### ISSUE THREAD

**Issue by metal450** [buchen/portfolio#1831]
This is a bug report for the *forum* (https://forum.portfolio-performance.info) rather than the software, so apologies if this is the incorrect place to post.

As the forum is almost entirely in German, it would be nice to be able to use Google Translate, so answers would be obtained without having to overburden with duplicate posts in English.  However, it seems to be incompatible with Google Translate.  Even in an "incognito window" (i.e. no browser plugins that could possibly be conflicting), entering a forum URL in Google Translate just yields an empty result.  Using the latest version of Firefox on Linux.  I've Googled around & found a few other examples of websites not being compatible with Google Translate, and the answer always comes down to it being some issue with the site itself.

(As a workaround, I've been trying to copy-paste the text from forum threads directly into Google Translate, but this is *MUCH* harder contextually, omits any embedded images or screenshots, leads to frequent misunderstanding, & is of course far more cumbersome).

So in summary: it would be great to have a look at why the forum isn't working with Google Translate :)

**Comment by chirlu** [comment]
It is indeed the completely wrong place. The proper place may be https://meta.discourse.org/t/add-a-get-option-to-enable-noscript-view-of-a-page/33823
As you can read there, adding `?_escape_fragment_` to the URL that you hand to Google Translate might help.

**Comment by metal450** [comment]
That did it!! Excellent, thanks, & closing :)
